doc/users/config_file.rst

```diff
@@ -14,93 +14,97 @@ Logging
 ~~~~~~~
 
 *workflow_level*
-	How detailed the logs regarding workflow should be (possible values:
-	``INFO`` and ``DEBUG``; default value: ``INFO``)
-*filemanip_level*
-	How detailed the logs regarding file operations (for example overwriting
-	warning) should be (possible values: ``INFO`` and ``DEBUG``; default value:
-	``INFO``)
+    How detailed the logs regarding workflow should be (possible values:
+    ``INFO`` and ``DEBUG``; default value: ``INFO``)
+*utils_level*
+    How detailed the logs regarding nipype utils, like file operations
+    (for example overwriting warning) or the resource profiler, should be
+    (possible values: ``INFO`` and ``DEBUG``; default value:
+    ``INFO``)
 *interface_level*
-	How detailed the logs regarding interface execution should be (possible
-	values: ``INFO`` and ``DEBUG``; default value: ``INFO``)
+    How detailed the logs regarding interface execution should be (possible
+    values: ``INFO`` and ``DEBUG``; default value: ``INFO``)
+*filemanip_level* (deprecated as of 0.13.2)
```

Comment:
since we are calling the next release 1.0, should this be deprecated as of `1.0` or `1.1`